### PR TITLE
Hi! Thank you for maintaining this awesome repository.  This PR adds a recent ICLR 2026 paper to the ### Other Merging Methods section.  Paper: Gradient-Sign Masking for Task Vector Transport Across Pre-Trained Models Method (GradFix): It introduces a novel framework to successfully transfer task vectors across different pre-trained models without additional fine-tuning. The adaptation is achieved by computing a few gradients at the target model and masking the source task vector, approximating the ideal gradient sign structure. +3  I made sure the formatting strictly follows the existing table structure. Let me know if any adjustments are needed.  Best regards!

### PR DESCRIPTION
**Pull request by fillo-rinaldi**
Hi! Thank you for maintaining this awesome repository.

This PR adds a recent ICLR 2026 paper to the ### Other Merging Methods section.

Paper: Gradient-Sign Masking for Task Vector Transport Across Pre-Trained Models Method (GradFix): It introduces a novel framework to successfully transfer task vectors across different pre-trained models without additional fine-tuning. The adaptation is achieved by computing a few gradients at the target model and masking the source task vector, approximating the ideal gradient sign structure.

I made sure the formatting strictly follows the existing table structure. Let me know if any adjustments are needed.

Best regards!

Filippo

### CODE REVIEW

**Comment by EnnengYang** [comment]
thanks!
